tasks/input_sql_file.yml

```diff
@@ -0,0 +1,35 @@
+- name: Verify if postgresql_password is set
+  assert:
+    that: postgresql_password is defined
+    fail_msg: >-
+      You must define postgresql_password variable to be able accessing
+      the database.
+
+- name: Copy SQL file on the host
+  copy:
+    src: "{{ postgresql_input_file }}"
+    dest: "/tmp/input.sql"
```

Comment:
Please use a `tempfile` module to create a temporary file and then copy into it. Like here:
https://github.com/linux-system-roles/mssql/blob/master/tasks/input_sql_file.yml#L17-L30

Quotes around "/tmp/input.sql" are not required